CMakeLists.txt

```diff
@@ -1221,6 +1225,17 @@ else()
         set_target_properties(${IMPL_TARGET} PROPERTIES PREFIX "")
     endif()
     target_link_libraries(${IMPL_TARGET} PRIVATE common ${LINKER_FLAGS} ${EXTRA_LIBS} ${MATH_LIB})
+
+    if(APPLE AND ALSOFT_OSX_FRAMEWORK)
+        set_target_properties(${IMPL_TARGET} PROPERTIES
+            FRAMEWORK TRUE
+            FRAMEWORK_VERSION C
+            MACOSX_FRAMEWORK_IDENTIFIER org.opena-soft.openal
+            MACOSX_FRAMEWORK_INFO_PLIST ${OpenAL_SOURCE_DIR}/info.plist
+            PUBLIC_HEADER include/AL/alc.h include/AL/al.h include/AL/alext.h 
+            XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer"
```

Comment:
Is this needed for command line based project generated by CMake? Xcode project should work fine without it.

Reply:
```sh
cd build
cmake -GXcode .. -DCMAKE_CXX_EXTENSIONS=OFF -DALSOFT_REQUIRE_COREAUDIO=ON  -DALSOFT_EMBED_HRTF_DATA=YES -DLIBTYPE=SHARED -DALSOFT_UTILS=OFF -DALSOFT_EXAMPLES=OFF -DALSOFT_INSTALL=OFF -DCMAKE_CXX_EXTENSIONS=OFF -DALSOFT_OSX_FRAMEWORK=ON
# open OpenAL xcode project, then build the OpenAL framework
```